features/machine/machine.feature

```diff
@@ -487,31 +487,36 @@ Feature: Machine features testing
   @admin
   @4.10 @4.9
   Scenario: Node labels and Affinity definition in PV should match
-    Given I have a project
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+    And I use the "openshift-machine-api" project
 
     # Create a pvc
     Given I obtain test data file "cloud/pvc-34718.yml"
     When I run the :create client command with:
       | f | pvc-34718.yml |
     Then the step should succeed
+    And admin ensures "pvc-cloud" pvc is deleted after scenario
 
     # Create a pod
     Given I obtain test data file "cloud/pod-34718.yml"
     When I run the :create client command with:
       | f | pod-34718.yml |
     Then the step should succeed
+    And admin ensures "task-pv-pod" pod is deleted after scenario 
 
     #Check node labels and affinity definition in PV are match
     Given the pod named "task-pv-pod" becomes ready
     And I use the "<%= pod.node_name %>" node
     And evaluation of `node.region` is stored in the :default_region clipboard
     And evaluation of `node.zone` is stored in the :default_zone clipboard
     When I run the :describe admin command with:
-      | resource | pv |
+      | resource | node                   |
+      | name     | <%= pod.node_name %>   |
     Then the step should succeed
     And the output should contain:
```

Comment:
@sunzhaohua2  , so it is safe to remove - 511,512 517-518 ?

Reply:
yes, I think so, we can remove all lines after line 510.

Reply:
Like
```
  Scenario: Node labels and Affinity definition in PV should match
    Given I have an IPI deployment
    And I switch to cluster admin pseudo user
    And I use the "openshift-machine-api" project

    # Create a pvc
    Given I obtain test data file "cloud/pvc-34718.yml"
    When I run the :create client command with:
      | f | pvc-34718.yml |
    Then the step should succeed
    And admin ensures "pvc-cloud" pvc is deleted after scenario

    # Create a pod
    Given I obtain test data file "cloud/pod-34718.yml"
    When I run the :create client command with:
      | f | pod-34718.yml |
    Then the step should succeed
    And admin ensures "task-pv-pod" pod is deleted after scenario 
    And the pod named "task-pv-pod" becomes ready
```

Reply:
but it didn't check Node labels and Affinity definition in PV should match, It's not consistent with the bug https://bugzilla.redhat.com/show_bug.cgi?id=1860128

Reply:
@huali9 based on - https://bugzilla.redhat.com/show_bug.cgi?id=1860128#c17 I feel this is fine.. 
@sunzhaohua2 WDYS ?

Reply:
I feel this is fine, if Node labels and Affinity definition in PV doesn't match, then pod will create failed. So we can simply check pod status, if pod failed, our case will fail, then we can check if there are issues about Affinity definition.

Reply:
yes, sorry, I hadn't finished reading this long bug...